rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -1324,4 +1457,146 @@ size_t Player::get_number_of_registered_on_play_msg_post_callbacks()
   return pimpl_->get_number_of_registered_on_play_msg_post_callbacks();
 }
 
+PlayerImpl::PlayerClient::PlayerClient(
+  std::shared_ptr<rclcpp::GenericClient> cli,
+  std::string service_name,
+  std::string service_event_type,
+  const rclcpp::Logger & logger)
+: client_(std::move(cli)),
+  service_name_(service_name),
```

Comment:
Comments from MichaelOrlov

```suggestion
  service_name_(std::move(service_name)),
```

Reply:
Updated